.github/workflows/canary_checks.yml

```diff
@@ -44,6 +44,10 @@ jobs:
       contents: read
     steps:
       - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # version 4.1.4
+        with:
+          # TODO temporarily pin to pre-pretty sandbox until we ship that
+          # Branch contains repo at 0cc2de3ae15f09a9eec5b9e0d54f0bae474c7aa9 plus changes to build step caching
```

Comment:
I had to pin to temporary branch, because old version didn't see build caching changes.

Temp branch is last canary-stable-main + https://github.com/aws-amplify/amplify-backend/commit/e250db527c675173c094f49729399aab37712225